forte/base_classes/mo_space_info.cc

```diff
@@ -421,9 +430,8 @@ std::pair<SpaceInfo, bool> MOSpaceInfo::read_mo_space(const std::string& space,
     return std::make_pair(space_info, read);
 }
 
-std::pair<SpaceInfo, bool>
-MOSpaceInfo::read_mo_space_from_map(const std::string& space,
-                                    const std::map<std::string, std::vector<size_t>>& mo_space_map) {
+std::pair<SpaceInfo, bool> MOSpaceInfo::read_mo_space_from_map(
+    const std::string& space, const std::map<std::string, std::vector<size_t>>& mo_space_map) {
```

Comment:
Please change the signature so that `mo_space_map` is `const` and the method is `const`.

Reply:
I am not sure the method can be made const.